src/sql/workbench/api/common/extHostDataProtocol.ts

```diff
@@ -291,19 +293,23 @@ export class ExtHostDataProtocol extends ExtHostDataProtocolShape {
 		if (this.uriTransformer) {
 			ownerUri = this._getTransformedUri(ownerUri, this.uriTransformer.transformIncoming);
 		}
-
+		this._perfProxy.$mark(`extTestMark at ${Date.now()}`);
```

Comment:
You mean to leave this in?

Reply:
No that was for testing, thanks for catching this I just cleaned it up.